src/create_tridiagonal.cpp

```diff
@@ -4,38 +4,50 @@
 // - lower diagonal: vector a, lenght n-1
 // - main diagonal:  vector d, lenght n
 // - upper diagonal: vector e, lenght n-1
-arma::mat create_tridiagonal(const arma::vec& a, const arma::vec& d, const arma::vec& e)
+arma::mat create_tridiagonal(const arma::vec& a, const arma::vec& d,const arma::vec& e)
```

Comment:
```suggestion
arma::mat create_tridiagonal(const arma::vec& a, const arma::vec& d, const arma::vec& e)
```
Satt inn mellomrom igjen